bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -2,6 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
+{% set_lang_files "foundation/base-resp" %}
```

Comment:
Actually, probably better to put them into index.lang and then set this lang file here?